### PR TITLE
Added signup page

### PR DESCRIPTION
**Pull request by kanavmittal314**
Created `SignupPage.java` class. Includes username and password.

### CODE REVIEW

**Comment by stellakaval** [comment]
You need to implement the `addAccount` method to finish this implementation. Great work so far.

File: SignupPage.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	private String username;
+	private String password;
+
+	public SignupPage(String username, String password){
+		this.username = username;
+		this.password = password;
+	}
+
+	public void addAccount() {
+		# TODO
```
Comment (stellakaval) [inline SignupPage.java]: You left this as a TODO, please complete so I can approve.

**Review by stellakaval — CHANGES_REQUESTED** [review]

File: SignupPage.java (hunk 1 above)
Comment (kanavmittal314) [reply SignupPage.java]: Done!

**Review by stellakaval — APPROVED** [review]
Great work, thanks.

**Pull request merged**